sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -160,7 +185,7 @@ class StreamExecution(
         sink.addBatch(nextBatch)
       }
 
-      batchRun = true
+      active = true
```

Comment:
`batchRun` and `shouldRun` were not the same.  I think something got mixed up here?